backend/api/views/main.py

```diff
@@ -24,26 +24,34 @@ def get_persons():
     return create_response(data={"persons": persons})
 
 
-# function that is called when you visit /census_response
+
+# Given:
+# date in database
+# (optional) tract_id in database
+#
+# Returns all response rates associated to that date (and id)
 @main.route("/census_response", methods=["GET"])
```

Comment:
wait, are we getting rid of the previous function of this endpoint? otherwise it could be useful to have added functionality instead of getting rid of the previous one
correct me if i'm reading this wrong though

Reply:
My endpoint adds to the previous functionality by allowing the addition of an optional tract_id parameter to further narrow the results. Before only a date parameter was allowed.